sdk/monitor/monitor-opentelemetry-exporter/samples/v1/javascript/package.json

```diff
@@ -33,7 +33,7 @@
     "@opentelemetry/api": "^1.0.1",
     "@opentelemetry/resources": "^0.22.0",
     "@opentelemetry/semantic-conventions": "^0.24.0",
-    "@opentelemetry/instrumentation": "0.22.0",
+    "@opentelemetry/instrumentation": "0.29.1",
```

Comment:
@[witemple-msft](https://github.com/witemple-msft) Should I be updating v1 samples at all? When I run the `build:samples` command it generates the samples in the v1-beta folder.